pkg/bbgo/activeorderbook.go

```diff
@@ -70,36 +70,36 @@ func (b *ActiveOrderBook) BindStream(stream types.Stream) {
 	stream.OnOrderUpdate(b.orderUpdateHandler)
 }
 
-func (b *ActiveOrderBook) waitClear(
-	ctx context.Context, order types.Order, waitTime, timeout time.Duration,
-) (bool, error) {
-	if !b.orders.Exists(order.OrderID) {
-		return true, nil
-	}
-
-	timeoutC := time.After(timeout)
-	for {
-		select {
-		case <-time.After(waitTime):
-		case <-b.C:
-		}
-
-		clear := !b.orders.Exists(order.OrderID)
-
-		select {
-		case <-timeoutC:
-			return clear, nil
-
-		case <-ctx.Done():
-			return clear, ctx.Err()
-
-		default:
-			if clear {
-				return clear, nil
-			}
-		}
-	}
-}
+// func (b *ActiveOrderBook) waitClear(
```

Comment:
Can you keep this function, and renmae it to waitOrderClear()

Reply:
should we public it, rename it to WaitOrderClear()? I think we should, so that strategies can take advantage of it

Reply:
I think we should only rename it when we are going to use it